test/new-e2e/tests/windows/service-test/startstop_test.go

```diff
@@ -776,3 +788,17 @@ func (s *baseStartStopSuite) sendHostMemoryMetrics(host *components.RemoteHost)
 		s.T().Logf("posted memory metrics")
 	}
 }
+
+// captureLiveKernelDump sends a commnad to the host to create a live kernel dump.
+func (s *baseStartStopSuite) captureLiveKernelDump(dumpDir string) {
+	getSubsystemCmd := `$ss = Get-CimInstance -ClassName MSFT_StorageSubSystem -Namespace Root\Microsoft\Windows\Storage`
+    createLiveDumpCmd := fmt.Sprintf(`Invoke-CimMethod -InputObject $ss -MethodName "GetDiagnosticInfo" -Arguments @{DestinationPath="%s"; IncludeLiveDump=$true}"`, dumpDir)
```

Comment:
it looks like this command puts the dump file in a subdirectory, `C:\dumps\localhost\LiveDump.dmp`, but the [dump collection isn't recursive](https://github.com/DataDog/datadog-agent/blob/0145beac3b8895657bd2bd2a726fdbe119a2556d/test/new-e2e/tests/windows/common/crashdump.go#L107-L110), so it won't be detected.

It seems like the command fails when run twice because the files already exist. So maybe it'd be better to run the command with a temp directory target and then copy only the dump file into the provided `dumpDir`.